### DISCUSSION TITLE
[5.0] Add Section, EntryType and Field Groups

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#14338] [Ideas]
I have a question and some thoughts regarding the organization of fields and sections in the admin area of Craft CMS. Previously, there was the capability to group fields and sections in the sidebar, which provided better clarity when dealing with a large number of items. This grouping feature has been removed, which initially seemed practical to me, but now I find myself missing the additional context provided by groups. Implementing a way to group fields using tags, allowing a field to be part of multiple groups, would be beneficial. It's worth noting positively that fields are not deleted when a group is removed.

**Discussion in Discord:**
- mattwilcox: mentioned that fields in version 5 no longer carry the same meaning as before, making grouping somewhat redundant. The advice is to name fields based on the type of content they hold, and then to drag them into your Entry Type (the context) as their own instances, rendering the old way of grouping a bit obsolete. It is anticipated that this will result in fewer fields being used.
- wemdo: agrees but notes that grouping for sections remains a valid point, as now everything including categories, Globals-Singles, and Tags are merged together. Currently, special characters are used to differentiate them into groups.
- Me: highlights the need for organizing fields, sections, and even EntryTypes in some kind of structure for the content admin, given that categories, globals, and sections are now all placed in the same area, making everything more "information rich."

**Summary and Proposal:**

Add some kind of grouping for 
- Fields
- Sections (more information than Craft 4)
- EntryTypes (more information than Craft 4)

One suggestion could be to introduce tag-based grouping, which would allow fields and sections to be flexibly organized into multiple groups. This would enable content administrators to manage their content more efficiently and with better organization.

**Comment by wmdhosting** [comment]
Sections need grouping..   perhaps  same with Icons..  as entry types..  

i do like this now.. 
![Snag_ba4d60e](https://github.com/craftcms/cms/assets/76176291/345b6fda-f96a-40f3-ba75-e4c2c75ab27f)

**Reply by wmdhosting** [reply]
Matrix entries types grouping..  would me great too :)

**Comment by wmdhosting** [comment]
Example  i use section field..    

so there icons can be shown too  and grouped by icons if posiblle 

So it eaiser to know what to select.. 
![Snag_bc1b1b0](https://github.com/craftcms/cms/assets/76176291/bf03dbbf-a0a7-4d31-8446-96d4f9026f4c)

**Comment by alexanderloewe** [comment]
@brandonkelly: Is something like this planned? If not, we have to use some kind of prefixes for the EntryTypes.

**Comment by maxstrebel** [comment]
+1 to have Groups (similar to Field Groups)
